### ISSUE TITLE
Incompatible with Oritech / owo-lib networking

### ISSUE THREAD

**Issue by Rearth** [qouteall/ImmersivePortalsMod#1646]
Adding immersive portals to a pack seems to complete break the Networking of Oritech, which uses the owo-lib networking modules. I have no idea how/why this happens, as Immersive Portals was the last mod I expected to modify the networking. It might be related to this mixin here, but that's just a wild guess: [MixinClientboundCustomPayloadPacket](https://github.com/iPortalTeam/ImmersivePortalsMod/blob/a9bd638e851fe5bf671500fde767d4a43ee04dd6/src/main/java/qouteall/imm_ptl/core/mixin/common/networking/MixinClientboundCustomPayloadPacket.java#L17)

**Comment by Valiant404-143** [comment]
Yeah, three months later Oritech still doesn't work with immersive portals, hopefully the Immersive portal devs fix this at some point.
